oak_runtime/src/node/http/mod.rs

```diff
@@ -483,11 +482,14 @@ impl Pipe {
 
 // TODO(#1279): Get the label from a JSON string instead of a binary label.
 fn get_oak_label(req: &HttpRequest) -> Result<Label, OakStatus> {
+    log::info!("headers: {:?}", &req.headers);
     match req.headers.get(oak_abi::OAK_LABEL_HTTP_KEY) {
```

Comment:
+1 to using different headers. Though using JSON seems like the more common case, it's seems not impossible that someone would want to send a Protobuf representation of labels over HTTP. 

For example in the introspection client we currently send Protobuf encoded data over a standard HTTP connection, since gRPC isn't compatible with web browsers.

Reply:
What do we do if the client provides both headers? Should we terminate with an error in those cases?

Reply:
Perhaps we should return the same error we return when no header is provided?